src/elements/content-explorer/Content.tsx

```diff
@@ -79,7 +81,15 @@ const Content = ({
                     {...rest}
                 />
             )}
-            {isMetadataViewV2Feature && !isViewEmpty && isMetadataBasedView && <MetadataView />}
+            {isMetadataViewV2Feature && isMetadataBasedView && (
```

Comment:
we need to exclude `isViewEmpty` from showing on metadata view empty?

Reply:
the metadata view component has its own empty view component, so when its empty here it feeds into the inner empty view component instead, unless you think we should use the empty view componet out here

Reply:
i meant that if the collection is empty, won't the regular ContentExplorer empty container AND the metadataviewV2 empty container BOTH be rendered at the same time?

Reply:
ah yes, youre right, i undid a change i made here, I have to add back the feature check on the emptyview render condition above this